ansible/group_vars/hypervisors

```diff
@@ -0,0 +1,38 @@
+---
+# Map physical network names to their source device. This can be either an
+# existing interface or an existing bridge.
+physnet_mappings: {}
+
+virtualisation_provider: libvirt
+
+system_requirements:
+  - python-pip
+  - python-virtualenv
+
+# Path to virtualenv used to install Python requirements. If a virtualenv does
+# not exist at this location, one will be created.
+virtualenv_path: ~/tenks-venv
+
+python_requirements:
+  - virtualbmc
```

Comment:
Will probably need to constrain this version at some point. The Ansible pip module doesn't make this easy though. You can pass it a requirements file, or a specific version, but not a single package with an upper version limit.

Reply:
I guess omething like `venv-requirements.txt` would be a good idea then?

Reply:
That would work. If you were to put the virtualbmc installation into a role, you could put it under a files/ directory in the role.